src/pages/TokenDetails/index.tsx

```diff
@@ -122,7 +117,7 @@ export default function TokenDetails() {
       {tokenAddress && (
         <>
           <RightPanel>
-            <Widget />
+            <SwapWidget provider={provider} />
```

Comment:
i know this is totally WIP, but should probably hideConnectionUI={true} & pass in jsonRpcUrlMap & routerUrl as well